tests/syntax/test_regions.py

```diff
@@ -102,26 +101,117 @@ def test_polyline_start():
     ego = sampleEgoFrom("""
         r = PolylineRegion([1@1, 3@-1, 6@2])
         pt = r.start
-        ego = Object at pt, facing pt.heading
+        ego = new Object at pt, facing pt.heading
     """)
-    assert tuple(ego.position) == pytest.approx((1, 1))
+    assert tuple(ego.position) == pytest.approx((1, 1, 0))
     assert ego.heading == pytest.approx(math.radians(-135))
 
 def test_polyline_end():
     ego = sampleEgoFrom("""
         r = PolylineRegion([1@1, 3@-1, 6@2])
         pt = r.end
-        ego = Object at pt, facing pt.heading
+        ego = new Object at pt, facing pt.heading
     """)
-    assert tuple(ego.position) == pytest.approx((6, 2))
+    assert tuple(ego.position) == pytest.approx((6, 2, 0))
     assert ego.heading == pytest.approx(math.radians(-45))
 
+# Mesh Regions
+def test_mesh_region_distribution():
+    sampleSceneFrom("""
+        position = (Range(-5,5), Range(-5,5), Range(-5,5))
+        radius = Range(1,5)
+        dimensions = (2*radius, 2*radius, 2*radius)
+        rotation = (Range(0,360), Range(0,360), Range(0,360))
+
+        region = SpheroidRegion(position=position, dimensions=dimensions, rotation=rotation)
+
+        ego = new Object in region
+    """, maxIterations=100)
+
+# View Regions
+def test_view_region_construction():
```

Comment:
Is this test just checking that the code doesn't crash for the different cases of view regions? It would be nice to break it into separate tests for each case and actually check that various points are visible/not visible in the expected way.

Reply:
Good point, I put adding this test on the list.